corehq/util/soft_assert/api.py

```diff
@@ -3,9 +3,18 @@
 from corehq.apps.hqwebapp.tasks import send_mail_async, mail_admins_async
 from corehq.util.global_request import get_request
 from corehq.util.soft_assert.core import SoftAssert
+from django.views.debug import get_exception_reporter_filter
 
 
 def _send_message(info, backend):
+    try:
+        request = get_request()
+        # de-sensitize requests that might have passwords
```

Comment:
yeah i think it's subjective. one downside of always calling it is that if anyone ever changes the behavior of `get_exception_reporter_filter` to disallow `None` it will unexpectedly break, though agree it is a bit more/messier code.